sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -425,6 +435,6 @@ export class RestrictionController extends BaseController {
       isActive: !!(
         institutionRestrictions && institutionRestrictions.length > 0
```

Comment:
This `institutionRestrictions && institutionRestrictions.length > 0` can be simplifed to `institutionRestrictions?.length`.

Reply:
It was not implemented as suggested. Is there a reason why? Why is it checking `<= 0`?